coreclr-1.0.0-beta5-11624/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:14.04
+
+ENV DNX_FEED https://www.myget.org/F/aspnetvnext/api/v2
+
+RUN apt-get update -qq \
+    && apt-get install -qqy libunwind8 libssl-dev curl unzip libtool automake build-essential
+
+# Install libuv for Kestrel from source code (binary is not in wheezy and one in jessie is still too old)
+RUN LIBUV_VERSION=1.4.2 \
+    && curl -sSL https://github.com/libuv/libuv/archive/v${LIBUV_VERSION}.tar.gz | tar zxfv - -C /usr/local/src \
+    && cd /usr/local/src/libuv-$LIBUV_VERSION \
+    && sh autogen.sh && ./configure && make && make install \
+    && rm -rf /usr/local/src/libuv-$LIBUV_VERSION \
+    && ldconfig
+
+RUN curl -sSL https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.sh | DNX_BRANCH=dev sh \
+    && bash -c "source $HOME/.dnx/dnvm/dnvm.sh \
+       && dnvm install 1.0.0-beta5-11624 -r coreclr -arch x64 -a default \
```

Comment:
We discussed this with @ahmetalpbalkan, and I believe (Ahmet please correct me if wrong) that unless something changes that results in re-generating the image produced by this dockerfile, the resultant image will stay the same (and thus not need to fetch dependencies) unless someone triggers an update for it.  My understanding was that given the runtime is already on the docker image used and all dependencies are in the tar.gz, the only way that we would break this setup from working would be if you unpinned the 11624 DNX and then someone regenerated the  coreclr-1.0.0-beta5-11624 image.